### ISSUE TITLE
Sorting

### ISSUE THREAD

**Issue by PVince81** [owncloud/customgroups#46]
Sorting is currently wrong and also done on the client side.

Once we add pagination, client-side sorting will not be possible any more so it needs to be done on the backend side.

- Groups list sort:
   - first: role (admin, then member)
   - second: group display name
- Members list sort:
   - first: role (admin, then member)
   - second: user display name (will become a headache until we have the account table to join on)

@pmaier1 FYI, I think these sort defaults make sense. I do not plan to make alternative sorts with clickable columns in this first version.

cc @felixheidecke

**Comment by PVince81** [comment]
Additionally: seems that backbone doesn't insert-sort after adding an item, needs to investigate.

**Comment by PVince81** [comment]
Sorting by displayname becomes more complex, that is unless we sort on PHP level...
Sorting on DB level would be nicer but we might want natural sort which is only possible on PHP or JS level...

**Comment by pmaier1** [comment]
> I think these sort defaults make sense.

Yes, looks good to me. Regarding your other comments I'm not quite sure if I get you right. If important, let's have a quick call.
